src/components/navbar.component.tsx

```diff
@@ -1,20 +1,28 @@
 import React from "react";
 import { navigate } from "@openmrs/esm-config";
-import { Location20, UserAvatarFilledAlt20 } from "@carbon/icons-react";
+import {
+  UserAvatarFilledAlt20,
+  Search20,
+  Add20,
+  Switcher20
+} from "@carbon/icons-react";
```

Comment:
we could use the default import for each icon for reducing bundle size, @FlorianRappl could best advice on it
```suggestion
import Switcher20 from "@carbon/icons-react/lib/switcher/20"
```

Reply:
True - should definitely be done as suggested by @donaldkibet

Reply:
Oh that's perfect thanks , will do this .